src/main/kotlin/io/github/mojira/arisa/domain/Attachment.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.mojira.arisa.domain
+
+import arrow.core.Either
+import java.util.Date
+
+data class Attachment(
+    val name: String,
+    val created: Date,
```

Comment:
That might have been inconsistent already, but we probably should change all of these to Instant for consistency.

Reply:
Indeed

Reply:
https://github.com/mojira/arisa-kt/issues/206